GSA-TTS/federal-website-standards
# Remove prototype label

# This should not be merged until the prototype label is ready to be removed from the site

## How to verify this change
1. Visit the [preview URL homepage](https://federalist-142078a8-f654-4daa-8f73-db9770b3ec7a.sites.pages.cloud.gov/preview/gsa-tts/federal-website-standards/caley/remove-prototype-label/) and check for the prototype label
